src/data/items.js

```diff
@@ -505,3 +507,25 @@ export const chocolateMilk = freeze({
   type: MILK,
   value: 80,
 })
+
+/**
+ * @property farmhand.module:items.chocolate
+ * @type {farmhand.item}
+ */
+export const chocolate = freeze({
+  id: 'chocolate',
+  name: 'Chocolate',
+  type: CHOCOLATE,
+  value: 500, // what's a good value?
+})
+
+/**
+ * @property farmhand.module:items.soyMilk
+ * @type {farmhand.item}
+ */
+export const soyMilk = freeze({
+  id: 'soy-milk',
+  name: 'Soy Milk',
+  type: SOY_MILK,
+  value: 100, // what's a good value?
+})
```

Comment:
This can be removed for the same reason, since it is being defined in `src/data/recipes.js`.